internal/Data/TypeRep/CacheMap.hs

```diff
@@ -125,12 +132,29 @@ cachedBinarySearch (Fingerprint (W64# a) (W64# b)) fpAs fpBs = inline (go 0#)
 -- Functions for testing and benchmarking
 ----------------------------------------------------------------------------
 
+-- | Existential wrapper around 'Typeable' indexed by @f@ type parameter.
+-- Useful for 'TypeRepMap' structure creation form list of 'TF's.
 data TF f where
     TF :: Typeable a => f a -> TF f
```

Comment:
I noticed that this constructor is not displayed properly with Haddock. Maybe you can create issue.

Reply:
Yes, that's probably something to do with reexporting `TF` from another module